src/index.ts

```diff
@@ -12,8 +12,10 @@ createActionsPlugin<AssistivePricingSettings, Env, SupportedEvents>(
     return run({ ...context, adapters: createAdapters(createClient(context.env.SUPABASE_URL, context.env.SUPABASE_KEY), context) });
   },
   {
+    // @ts-expect-error types match
```

Comment:
What's this for

Reply:
The `Manifest` type is not exported yet by the kernel, should be fixed once https://github.com/ubiquity-os/ubiquity-os-kernel/pull/169 is merged, in the meantime that creates an error so I needed to silence it.